security/org-account-context/main.tf

```diff
@@ -518,6 +518,17 @@ module "config_local_2" {
   }
 }
 
+resource "aws_account_alternate_contact" "security" {
+  count                  = var.harden ? 1 : 0
+  alternate_contact_type = "SECURITY"
+  name                   = "Cloud Engineering"
+  title                  = "Cloud Engineering"
+  email_address          = "cloud.engineering+${var.name}@dfds.com"
```

Comment:
This is a public repo, and in order to avoid unnecessary spam, I think this should be a variable with no default values.

Reply:
Make sense, addressed.